blog/en-us/spring-boot-dubbo-start-stop-analysis.md

```diff
@@ -0,0 +1,223 @@
+# Source code analysis of spring-boot+Dubbo App start and stop
+
+### Introduction
+
+[Dubbo Spring Boot](https://github.com/apache/incubator-dubbo-spring-boot-project) project is dedicated to simplifying the development of the Dubbo RPC framework in the Spring Boot application. It also integrates the feature of Spring Boot:
+
+- [Autoconfigure](https://github.com/apache/incubator-dubbo-spring-boot-project/blob/master/dubbo-spring-boot-autoconfigure) (ex: Annotation driver, Autoconfigure, etc.)
+- [Production-Ready](https://github.com/apache/incubator-dubbo-spring-boot-project/blob/master/dubbo-spring-boot-actuator) (ex: Security, Healthy check, Externalize configuration, etc.)
+
+### The analysis of DubboConsumer startup
+
+Have you ever thought about a problem? `DubboConsumerDemo` application in `incubator-dubbo-spring-boot-project` just have one line of code, why not exit directly when the `main` method is executed?
```

Comment:
Have you ever thought about this : since the `DubboConsumerDemo` application in `incubator-dubbo-spring-boot-project` has just one line of code, why not just exit directly when the `main` method is executed?

Reply:
Have you ever thought about this : since the DubboConsumerDemo application in incubator-dubbo-spring-boot-project has only one line of code, why not just exit directly when the main method is executed?